Tests/CommonHelpers/CMakeLists.txt

```diff
@@ -15,5 +17,5 @@ target_include_directories(
   PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
 target_link_libraries(
   ActsTestsCommonHelpers
-  INTERFACE ActsCore
```

Comment:
Why the target type change?

Reply:
Because `ActsTestsCommonHelpers` is not an interface library. I think it was previously and we forgot to fix this.

Reply:
ok